config-linux.md

```diff
@@ -438,7 +438,7 @@ The following parameters can be specified to setup the controller:
 
 * **`priorities`** *(array, OPTIONAL)* - specifies a list of objects of the priorities assigned to traffic originating from processes in the group and egressing the system on various interfaces.
   The following parameters can be specified per-priority:
-    * **`name`** *(string, REQUIRED)* - interface name
+    * **`name`** *(string, REQUIRED)* - interface name in [runtime network namespace](glossary.md#runtime-namespace)
```

Comment:
It appears the rest of this file is using footer links (`[link text][link ID]` with then a `[link ID]: URL` at the bottom of the file such as `[container-namespace2]: glossary.md#container_namespace`); should we stay consistent on that?